[2024-10-18T00:00:09.842+0000] {processor.py:186} INFO - Started process (PID=5855) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:00:09.844+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:00:09.847+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:00:09.847+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:00:09.869+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:00:09.864+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:00:09.872+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:00:09.912+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.079 seconds
[2024-10-18T00:00:40.040+0000] {processor.py:186} INFO - Started process (PID=5866) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:00:40.041+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:00:40.044+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:00:40.043+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:00:40.065+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:00:40.061+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:00:40.067+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:00:40.102+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.076 seconds
[2024-10-18T00:01:10.478+0000] {processor.py:186} INFO - Started process (PID=5877) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:01:10.491+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:01:10.497+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:01:10.495+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:01:10.524+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:01:10.518+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:01:10.526+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:01:10.562+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.131 seconds
[2024-10-18T00:01:40.855+0000] {processor.py:186} INFO - Started process (PID=5888) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:01:40.858+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:01:40.865+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:01:40.862+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:01:40.921+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:01:40.910+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:01:40.926+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:01:40.985+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.168 seconds
[2024-10-18T00:02:11.260+0000] {processor.py:186} INFO - Started process (PID=5899) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:02:11.261+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:02:11.263+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:02:11.263+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:02:11.287+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:02:11.282+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:02:11.289+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:02:11.325+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.074 seconds
[2024-10-18T00:02:41.468+0000] {processor.py:186} INFO - Started process (PID=5910) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:02:41.471+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:02:41.475+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:02:41.474+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:02:41.518+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:02:41.509+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:02:41.521+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:02:41.579+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.132 seconds
[2024-10-18T00:03:11.677+0000] {processor.py:186} INFO - Started process (PID=5921) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:03:11.678+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:03:11.682+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:03:11.681+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:03:11.706+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:03:11.700+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:03:11.708+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:03:11.744+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.078 seconds
[2024-10-18T00:03:42.454+0000] {processor.py:186} INFO - Started process (PID=5931) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:03:42.457+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:03:42.463+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:03:42.462+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:03:42.488+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:03:42.483+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:03:42.490+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:03:42.524+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.099 seconds
[2024-10-18T00:04:12.784+0000] {processor.py:186} INFO - Started process (PID=5943) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:04:12.787+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:04:12.791+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:04:12.790+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:04:12.845+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:04:12.833+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:04:12.850+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:04:12.915+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.151 seconds
[2024-10-18T00:04:43.485+0000] {processor.py:186} INFO - Started process (PID=5953) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:04:43.487+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:04:43.489+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:04:43.489+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:04:43.511+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:04:43.504+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:04:43.514+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:04:43.539+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.074 seconds
[2024-10-18T00:05:13.871+0000] {processor.py:186} INFO - Started process (PID=5964) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:05:13.875+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:05:13.879+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:05:13.878+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:05:13.935+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:05:13.922+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:05:13.940+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:05:14.000+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.150 seconds
[2024-10-18T00:05:44.277+0000] {processor.py:186} INFO - Started process (PID=5975) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:05:44.279+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:05:44.282+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:05:44.282+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:05:44.313+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:05:44.306+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:05:44.315+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:05:44.357+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.097 seconds
[2024-10-18T00:06:14.481+0000] {processor.py:186} INFO - Started process (PID=5987) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:06:14.484+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:06:14.489+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:06:14.488+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:06:14.541+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:06:14.532+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:06:14.544+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:06:14.625+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.179 seconds
[2024-10-18T00:06:44.771+0000] {processor.py:186} INFO - Started process (PID=5998) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:06:44.773+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:06:44.776+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:06:44.775+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:06:44.801+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:06:44.796+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:06:44.804+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:06:44.847+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.088 seconds
[2024-10-18T00:07:15.055+0000] {processor.py:186} INFO - Started process (PID=6009) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:07:15.060+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:07:15.065+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:07:15.064+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:07:15.097+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:07:15.090+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:07:15.100+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:07:15.134+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.117 seconds
[2024-10-18T00:07:45.318+0000] {processor.py:186} INFO - Started process (PID=6020) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:07:45.322+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:07:45.334+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:07:45.332+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:07:45.403+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:07:45.397+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:07:45.406+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:07:45.455+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.173 seconds
[2024-10-18T00:08:16.563+0000] {processor.py:186} INFO - Started process (PID=6031) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:08:16.570+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:08:16.579+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:08:16.578+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:08:16.661+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:08:16.642+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:08:16.667+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:08:16.763+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.238 seconds
[2024-10-18T00:08:47.454+0000] {processor.py:186} INFO - Started process (PID=6042) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:08:47.458+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:08:47.463+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:08:47.462+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:08:47.494+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:08:47.486+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:08:47.497+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:08:47.533+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.124 seconds
[2024-10-18T00:09:17.743+0000] {processor.py:186} INFO - Started process (PID=6052) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:09:17.746+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:09:17.751+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:09:17.749+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:09:17.811+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:09:17.798+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:09:17.816+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:09:17.879+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.180 seconds
[2024-10-18T00:09:48.028+0000] {processor.py:186} INFO - Started process (PID=6063) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:09:48.032+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:09:48.035+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:09:48.034+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:09:48.070+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:09:48.061+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:09:48.073+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:09:48.115+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.110 seconds
[2024-10-18T00:10:18.954+0000] {processor.py:186} INFO - Started process (PID=6074) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:10:18.956+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:10:18.958+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:10:18.958+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:10:18.984+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:10:18.979+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:10:18.987+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:10:19.019+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.073 seconds
[2024-10-18T00:10:49.543+0000] {processor.py:186} INFO - Started process (PID=6085) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:10:49.546+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:10:49.549+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:10:49.548+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:10:49.577+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:10:49.570+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:10:49.580+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:10:49.629+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.099 seconds
[2024-10-18T00:11:19.824+0000] {processor.py:186} INFO - Started process (PID=6096) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:11:19.833+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:11:19.841+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:11:19.840+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:11:19.902+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:11:19.888+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:11:19.906+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:11:19.969+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.187 seconds
[2024-10-18T00:11:50.248+0000] {processor.py:186} INFO - Started process (PID=6107) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:11:50.250+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:11:50.255+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:11:50.254+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:11:50.285+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:11:50.279+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:11:50.288+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:11:50.331+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.098 seconds
[2024-10-18T00:12:20.653+0000] {processor.py:186} INFO - Started process (PID=6118) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:12:20.654+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:12:20.656+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:12:20.655+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:12:20.682+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:12:20.677+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:12:20.684+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:12:20.718+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.079 seconds
[2024-10-18T00:12:50.933+0000] {processor.py:186} INFO - Started process (PID=6129) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:12:50.935+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:12:50.936+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:12:50.936+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:12:50.956+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:12:50.951+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:12:50.958+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:12:50.993+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.071 seconds
[2024-10-18T00:13:21.142+0000] {processor.py:186} INFO - Started process (PID=6140) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:13:21.143+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:13:21.145+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:13:21.145+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:13:21.168+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:13:21.164+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:13:21.170+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:13:21.203+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.070 seconds
[2024-10-18T00:13:52.026+0000] {processor.py:186} INFO - Started process (PID=6151) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:13:52.027+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:13:52.029+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:13:52.029+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:13:52.048+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:13:52.044+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:13:52.049+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:13:52.085+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.068 seconds
[2024-10-18T00:14:22.780+0000] {processor.py:186} INFO - Started process (PID=6162) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:14:22.781+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:14:22.783+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:14:22.783+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:14:22.804+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:14:22.799+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:14:22.806+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:14:22.841+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.070 seconds
[2024-10-18T00:14:53.191+0000] {processor.py:186} INFO - Started process (PID=6173) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:14:53.193+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:14:53.197+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:14:53.197+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:14:53.226+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:14:53.220+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:14:53.229+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:14:53.302+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.123 seconds
[2024-10-18T00:15:23.598+0000] {processor.py:186} INFO - Started process (PID=6184) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:15:23.599+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:15:23.603+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:15:23.603+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:15:23.625+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:15:23.621+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:15:23.627+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:15:23.659+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.070 seconds
[2024-10-18T00:15:54.570+0000] {processor.py:186} INFO - Started process (PID=6195) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:15:54.573+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:15:54.576+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:15:54.575+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:15:54.609+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:15:54.602+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:15:54.612+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:15:54.653+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.105 seconds
[2024-10-18T00:16:25.094+0000] {processor.py:186} INFO - Started process (PID=6206) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:16:25.096+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:16:25.099+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:16:25.098+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:16:25.129+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:16:25.124+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:16:25.132+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:16:25.164+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.083 seconds
[2024-10-18T00:16:55.592+0000] {processor.py:186} INFO - Started process (PID=6217) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:16:55.594+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:16:55.596+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:16:55.596+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:16:55.619+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:16:55.615+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:16:55.621+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:16:55.658+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.074 seconds
[2024-10-18T00:17:26.440+0000] {processor.py:186} INFO - Started process (PID=6228) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:17:26.441+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:17:26.443+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:17:26.443+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:17:26.465+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:17:26.461+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:17:26.467+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:17:26.494+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.064 seconds
[2024-10-18T00:17:57.078+0000] {processor.py:186} INFO - Started process (PID=6240) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:17:57.080+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:17:57.082+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:17:57.082+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:17:57.103+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:17:57.099+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:17:57.105+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:17:57.140+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.073 seconds
[2024-10-18T00:18:27.686+0000] {processor.py:186} INFO - Started process (PID=6251) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:18:27.688+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:18:27.690+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:18:27.690+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:18:27.729+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:18:27.718+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:18:27.732+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:18:27.780+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.105 seconds
[2024-10-18T00:18:57.949+0000] {processor.py:186} INFO - Started process (PID=6262) to work on /opt/airflow/dags/bas.py
[2024-10-18T00:18:57.951+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T00:18:57.953+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:18:57.953+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T00:18:58.026+0000] {logging_mixin.py:190} INFO - [2024-10-18T00:18:58.020+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T00:18:58.029+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T00:18:58.129+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.194 seconds
[2024-10-18T04:58:13.260+0000] {processor.py:186} INFO - Started process (PID=6278) to work on /opt/airflow/dags/bas.py
[2024-10-18T04:58:13.299+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T04:58:13.317+0000] {logging_mixin.py:190} INFO - [2024-10-18T04:58:13.316+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T04:58:14.978+0000] {logging_mixin.py:190} INFO - [2024-10-18T04:58:14.479+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T04:58:15.154+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T04:58:15.289+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 2.053 seconds
[2024-10-18T04:58:49.250+0000] {processor.py:186} INFO - Started process (PID=6287) to work on /opt/airflow/dags/bas.py
[2024-10-18T04:58:49.253+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T04:58:49.255+0000] {logging_mixin.py:190} INFO - [2024-10-18T04:58:49.255+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T04:58:49.283+0000] {logging_mixin.py:190} INFO - [2024-10-18T04:58:49.277+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T04:58:49.301+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T04:58:49.333+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.097 seconds
[2024-10-18T04:59:19.777+0000] {processor.py:186} INFO - Started process (PID=6298) to work on /opt/airflow/dags/bas.py
[2024-10-18T04:59:19.780+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T04:59:19.783+0000] {logging_mixin.py:190} INFO - [2024-10-18T04:59:19.782+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T04:59:19.824+0000] {logging_mixin.py:190} INFO - [2024-10-18T04:59:19.817+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T04:59:19.827+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T04:59:19.879+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.117 seconds
[2024-10-18T04:59:50.142+0000] {processor.py:186} INFO - Started process (PID=6308) to work on /opt/airflow/dags/bas.py
[2024-10-18T04:59:50.144+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T04:59:50.148+0000] {logging_mixin.py:190} INFO - [2024-10-18T04:59:50.147+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T04:59:50.207+0000] {logging_mixin.py:190} INFO - [2024-10-18T04:59:50.190+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T04:59:50.211+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T04:59:50.307+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.183 seconds
[2024-10-18T05:00:21.182+0000] {processor.py:186} INFO - Started process (PID=6318) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:00:21.188+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:00:21.199+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:00:21.198+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:00:21.477+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:00:21.439+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:00:21.666+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:00:21.812+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.688 seconds
[2024-10-18T05:00:52.606+0000] {processor.py:186} INFO - Started process (PID=6329) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:00:52.610+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:00:52.622+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:00:52.621+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:00:52.698+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:00:52.684+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:00:52.701+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:00:52.790+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.205 seconds
[2024-10-18T05:01:23.578+0000] {processor.py:186} INFO - Started process (PID=6340) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:01:23.589+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:01:23.592+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:01:23.591+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:01:23.628+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:01:23.621+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:01:23.637+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:01:23.713+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.154 seconds
[2024-10-18T05:01:53.861+0000] {processor.py:186} INFO - Started process (PID=6351) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:01:53.863+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:01:53.874+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:01:53.874+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:01:53.915+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:01:53.908+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:01:53.918+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:01:53.996+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.159 seconds
[2024-10-18T05:02:24.564+0000] {processor.py:186} INFO - Started process (PID=6363) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:02:24.566+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:02:24.570+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:02:24.569+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:02:24.613+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:02:24.607+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:02:24.616+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:02:24.657+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.116 seconds
[2024-10-18T05:02:55.779+0000] {processor.py:186} INFO - Started process (PID=6374) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:02:55.783+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:02:55.811+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:02:55.799+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:02:55.906+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:02:55.889+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:02:55.915+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:02:56.067+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.305 seconds
[2024-10-18T05:03:26.387+0000] {processor.py:186} INFO - Started process (PID=6385) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:03:26.393+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:03:26.404+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:03:26.402+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:03:26.552+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:03:26.518+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:03:26.561+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:03:26.653+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.307 seconds
[2024-10-18T05:03:56.950+0000] {processor.py:186} INFO - Started process (PID=6397) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:03:56.956+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:03:56.965+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:03:56.965+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:03:56.992+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:03:56.987+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:03:56.994+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:03:57.064+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.135 seconds
[2024-10-18T05:04:28.018+0000] {processor.py:186} INFO - Started process (PID=6408) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:04:28.020+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:04:28.024+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:04:28.023+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:04:28.055+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:04:28.049+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:04:28.058+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:04:28.099+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.095 seconds
[2024-10-18T05:04:58.457+0000] {processor.py:186} INFO - Started process (PID=6419) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:04:58.459+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:04:58.462+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:04:58.461+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:04:58.487+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:04:58.480+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:04:58.490+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:04:58.532+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.087 seconds
[2024-10-18T05:05:29.440+0000] {processor.py:186} INFO - Started process (PID=6430) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:05:29.442+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:05:29.454+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:05:29.453+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:05:29.493+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:05:29.486+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:05:29.496+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:05:29.580+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.166 seconds
[2024-10-18T05:06:00.066+0000] {processor.py:186} INFO - Started process (PID=6441) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:06:00.067+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:06:00.070+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:06:00.070+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:06:00.100+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:06:00.094+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:06:00.105+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:06:00.155+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.103 seconds
[2024-10-18T05:06:30.782+0000] {processor.py:186} INFO - Started process (PID=6452) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:06:30.784+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:06:30.787+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:06:30.787+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:06:30.812+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:06:30.805+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:06:30.815+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:06:30.860+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.089 seconds
[2024-10-18T05:07:01.454+0000] {processor.py:186} INFO - Started process (PID=6463) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:07:01.456+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:07:01.469+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:07:01.468+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:07:01.500+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:07:01.493+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:07:01.516+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:07:01.587+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.156 seconds
[2024-10-18T05:07:32.162+0000] {processor.py:186} INFO - Started process (PID=6473) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:07:32.164+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:07:32.168+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:07:32.167+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:07:32.194+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:07:32.188+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:07:32.196+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:07:32.232+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.083 seconds
[2024-10-18T05:08:02.909+0000] {processor.py:186} INFO - Started process (PID=6483) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:08:02.916+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:08:02.922+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:08:02.922+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:08:02.958+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:08:02.947+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:08:02.961+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:08:03.030+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.142 seconds
[2024-10-18T05:08:33.759+0000] {processor.py:186} INFO - Started process (PID=6494) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:08:33.762+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:08:33.764+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:08:33.764+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:08:33.792+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:08:33.785+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:08:33.794+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:08:33.840+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.093 seconds
[2024-10-18T05:09:04.243+0000] {processor.py:186} INFO - Started process (PID=6505) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:09:04.245+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:09:04.247+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:09:04.246+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:09:04.272+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:09:04.266+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:09:04.274+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:09:04.306+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.077 seconds
[2024-10-18T05:09:35.152+0000] {processor.py:186} INFO - Started process (PID=6515) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:09:35.155+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:09:35.162+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:09:35.161+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:09:35.194+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:09:35.185+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:09:35.197+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:09:35.248+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.115 seconds
[2024-10-18T05:10:05.933+0000] {processor.py:186} INFO - Started process (PID=6526) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:10:05.935+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:10:05.941+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:10:05.940+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:10:05.976+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:10:05.969+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:10:05.979+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:10:06.025+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.107 seconds
[2024-10-18T05:10:36.636+0000] {processor.py:186} INFO - Started process (PID=6537) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:10:36.637+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:10:36.639+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:10:36.639+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:10:36.666+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:10:36.660+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:10:36.668+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:10:36.712+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.088 seconds
[2024-10-18T05:11:06.825+0000] {processor.py:186} INFO - Started process (PID=6548) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:11:06.827+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:11:06.834+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:11:06.833+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:11:06.882+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:11:06.875+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:11:06.885+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:11:06.926+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.123 seconds
[2024-10-18T05:11:37.005+0000] {processor.py:186} INFO - Started process (PID=6560) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:11:37.007+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:11:37.009+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:11:37.009+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:11:37.034+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:11:37.029+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:11:37.036+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:11:37.062+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.068 seconds
[2024-10-18T05:12:07.599+0000] {processor.py:186} INFO - Started process (PID=6572) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:12:07.600+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:12:07.602+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:12:07.602+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:12:07.642+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:12:07.633+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:12:07.644+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:12:07.685+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.104 seconds
[2024-10-18T05:12:38.374+0000] {processor.py:186} INFO - Started process (PID=6583) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:12:38.376+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:12:38.380+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:12:38.380+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:12:38.414+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:12:38.408+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:12:38.417+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:12:38.452+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.089 seconds
[2024-10-18T05:13:08.675+0000] {processor.py:186} INFO - Started process (PID=6594) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:13:08.677+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:13:08.679+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:13:08.679+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:13:08.705+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:13:08.699+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:13:08.709+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:13:08.746+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.083 seconds
[2024-10-18T05:13:39.327+0000] {processor.py:186} INFO - Started process (PID=6605) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:13:39.329+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:13:39.331+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:13:39.331+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:13:39.357+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:13:39.351+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:13:39.362+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:13:39.417+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.113 seconds
[2024-10-18T05:14:10.369+0000] {processor.py:186} INFO - Started process (PID=6616) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:14:10.372+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:14:10.376+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:14:10.375+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:14:10.411+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:14:10.404+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:14:10.413+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:14:10.450+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.135 seconds
[2024-10-18T05:14:40.655+0000] {processor.py:186} INFO - Started process (PID=6627) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:14:40.657+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:14:40.660+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:14:40.659+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:14:40.683+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:14:40.677+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:14:40.686+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:14:40.737+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.100 seconds
[2024-10-18T05:15:10.858+0000] {processor.py:186} INFO - Started process (PID=6637) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:15:10.860+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:15:10.863+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:15:10.862+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:15:10.888+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:15:10.882+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:15:10.890+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:15:10.934+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.089 seconds
[2024-10-18T05:15:41.270+0000] {processor.py:186} INFO - Started process (PID=6648) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:15:41.272+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:15:41.275+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:15:41.274+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:15:41.308+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:15:41.301+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:15:41.311+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:15:41.361+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.119 seconds
[2024-10-18T05:16:11.832+0000] {processor.py:186} INFO - Started process (PID=6659) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:16:11.836+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:16:11.841+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:16:11.840+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:16:11.900+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:16:11.877+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:16:11.911+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:16:12.029+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.215 seconds
[2024-10-18T05:16:42.771+0000] {processor.py:186} INFO - Started process (PID=6670) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:16:42.774+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:16:42.779+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:16:42.778+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:16:42.826+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:16:42.815+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:16:42.830+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:16:42.898+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.149 seconds
[2024-10-18T05:17:13.666+0000] {processor.py:186} INFO - Started process (PID=6679) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:17:13.668+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:17:13.673+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:17:13.672+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:17:13.698+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:17:13.693+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:17:13.701+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:17:13.737+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.084 seconds
[2024-10-18T05:17:44.029+0000] {processor.py:186} INFO - Started process (PID=6689) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:17:44.034+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:17:44.044+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:17:44.042+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:17:44.106+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:17:44.093+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:17:44.112+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:17:44.174+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.172 seconds
[2024-10-18T05:18:14.609+0000] {processor.py:186} INFO - Started process (PID=6700) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:18:14.612+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:18:14.616+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:18:14.615+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:18:14.685+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:18:14.678+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:18:14.688+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:18:14.760+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.189 seconds
[2024-10-18T05:18:45.177+0000] {processor.py:186} INFO - Started process (PID=6711) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:18:45.186+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:18:45.197+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:18:45.195+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:18:45.245+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:18:45.234+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:18:45.250+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:18:45.339+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.211 seconds
[2024-10-18T05:19:16.206+0000] {processor.py:186} INFO - Started process (PID=6729) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:19:16.207+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:19:16.209+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:19:16.209+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:19:16.240+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:19:16.234+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:19:16.242+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:19:16.274+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.084 seconds
[2024-10-18T05:19:46.405+0000] {processor.py:186} INFO - Started process (PID=6741) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:19:46.407+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:19:46.409+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:19:46.408+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:19:46.430+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:19:46.425+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:19:46.433+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:19:46.490+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.095 seconds
[2024-10-18T05:20:16.800+0000] {processor.py:186} INFO - Started process (PID=6753) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:20:16.819+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:20:16.824+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:20:16.824+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:20:16.898+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:20:16.868+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:20:16.902+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:20:16.990+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.213 seconds
[2024-10-18T05:20:47.226+0000] {processor.py:186} INFO - Started process (PID=6764) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:20:47.228+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:20:47.231+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:20:47.230+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:20:47.250+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:20:47.246+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:20:47.252+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:20:47.293+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.081 seconds
[2024-10-18T05:21:17.714+0000] {processor.py:186} INFO - Started process (PID=6775) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:21:17.716+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:21:17.720+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:21:17.719+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:21:17.751+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:21:17.744+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:21:17.753+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:21:17.815+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.115 seconds
[2024-10-18T05:21:48.040+0000] {processor.py:186} INFO - Started process (PID=6786) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:21:48.044+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:21:48.049+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:21:48.048+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:21:48.097+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:21:48.086+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:21:48.102+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:21:48.175+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.165 seconds
[2024-10-18T05:22:18.960+0000] {processor.py:186} INFO - Started process (PID=6797) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:22:18.964+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:22:18.967+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:22:18.966+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:22:18.997+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:22:18.991+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:22:19.000+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:22:19.039+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.107 seconds
[2024-10-18T05:22:49.767+0000] {processor.py:186} INFO - Started process (PID=6808) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:22:49.777+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:22:49.782+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:22:49.782+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:22:49.881+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:22:49.865+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:22:49.896+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:22:49.990+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.256 seconds
[2024-10-18T05:23:21.227+0000] {processor.py:186} INFO - Started process (PID=6819) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:23:21.230+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:23:21.235+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:23:21.234+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:23:21.275+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:23:21.267+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:23:21.278+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:23:21.344+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.144 seconds
[2024-10-18T05:23:51.686+0000] {processor.py:186} INFO - Started process (PID=6830) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:23:51.688+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:23:51.690+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:23:51.690+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:23:51.713+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:23:51.708+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:23:51.716+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:23:51.757+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.085 seconds
[2024-10-18T05:24:22.131+0000] {processor.py:186} INFO - Started process (PID=6840) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:24:22.133+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:24:22.136+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:24:22.135+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:24:22.168+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:24:22.160+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:24:22.171+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:24:22.218+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.115 seconds
[2024-10-18T05:24:52.357+0000] {processor.py:186} INFO - Started process (PID=6851) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:24:52.363+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:24:52.372+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:24:52.371+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:24:52.402+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:24:52.395+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:24:52.404+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:24:52.503+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.171 seconds
[2024-10-18T05:25:23.335+0000] {processor.py:186} INFO - Started process (PID=6862) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:25:23.336+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:25:23.339+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:25:23.339+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:25:23.369+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:25:23.363+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:25:23.377+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:25:23.453+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.142 seconds
[2024-10-18T05:25:53.786+0000] {processor.py:186} INFO - Started process (PID=6872) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:25:53.789+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:25:53.793+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:25:53.792+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:25:53.829+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:25:53.820+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:25:53.832+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:25:53.879+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.118 seconds
[2024-10-18T05:26:24.429+0000] {processor.py:186} INFO - Started process (PID=6883) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:26:24.431+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:26:24.436+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:26:24.434+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:26:24.473+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:26:24.467+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:26:24.476+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:26:24.514+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.102 seconds
[2024-10-18T05:26:54.652+0000] {processor.py:186} INFO - Started process (PID=6894) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:26:54.654+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:26:54.658+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:26:54.657+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:26:54.691+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:26:54.682+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:26:54.695+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:26:54.776+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.143 seconds
[2024-10-18T05:27:25.261+0000] {processor.py:186} INFO - Started process (PID=6905) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:27:25.264+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:27:25.267+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:27:25.267+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:27:25.297+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:27:25.292+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:27:25.299+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:27:25.330+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.100 seconds
[2024-10-18T05:27:55.748+0000] {processor.py:186} INFO - Started process (PID=6916) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:27:55.753+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:27:55.763+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:27:55.762+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:27:55.800+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:27:55.794+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:27:55.803+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:27:55.874+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.148 seconds
[2024-10-18T05:28:26.196+0000] {processor.py:186} INFO - Started process (PID=6927) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:28:26.198+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:28:26.200+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:28:26.200+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:28:26.234+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:28:26.225+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:28:26.237+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:28:26.285+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.103 seconds
[2024-10-18T05:28:57.368+0000] {processor.py:186} INFO - Started process (PID=6937) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:28:57.393+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:28:57.418+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:28:57.417+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:28:57.516+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:28:57.496+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:28:57.520+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:28:57.657+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.314 seconds
[2024-10-18T05:29:28.340+0000] {processor.py:186} INFO - Started process (PID=6948) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:29:28.345+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:29:28.355+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:29:28.353+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:29:28.433+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:29:28.417+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:29:28.440+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:29:28.543+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.238 seconds
[2024-10-18T05:29:59.548+0000] {processor.py:186} INFO - Started process (PID=6959) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:29:59.550+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:29:59.553+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:29:59.552+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:29:59.581+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:29:59.575+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:29:59.584+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:29:59.625+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.122 seconds
[2024-10-18T05:30:29.955+0000] {processor.py:186} INFO - Started process (PID=6970) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:30:29.961+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:30:29.966+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:30:29.965+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:30:30.052+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:30:30.039+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:30:30.055+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:30:30.198+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.282 seconds
[2024-10-18T05:31:00.466+0000] {processor.py:186} INFO - Started process (PID=6981) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:31:00.469+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:31:00.474+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:31:00.472+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:31:00.504+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:31:00.497+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:31:00.507+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:31:00.546+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.105 seconds
[2024-10-18T05:31:30.716+0000] {processor.py:186} INFO - Started process (PID=6992) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:31:30.718+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:31:30.722+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:31:30.721+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:31:30.762+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:31:30.754+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:31:30.766+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:31:30.817+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.118 seconds
[2024-10-18T05:32:00.940+0000] {processor.py:186} INFO - Started process (PID=7003) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:32:00.941+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:32:00.943+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:32:00.943+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:32:00.967+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:32:00.961+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:32:00.971+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:32:01.007+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.077 seconds
[2024-10-18T05:32:31.079+0000] {processor.py:186} INFO - Started process (PID=7013) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:32:31.080+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:32:31.083+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:32:31.082+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:32:31.106+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:32:31.101+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:32:31.108+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:32:31.154+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.088 seconds
[2024-10-18T05:33:01.427+0000] {processor.py:186} INFO - Started process (PID=7022) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:33:01.428+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:33:01.432+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:33:01.431+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:33:01.465+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:33:01.460+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:33:01.467+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:33:01.497+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.095 seconds
[2024-10-18T05:33:31.908+0000] {processor.py:186} INFO - Started process (PID=7034) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:33:31.910+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:33:31.913+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:33:31.912+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:33:31.961+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:33:31.954+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:33:31.967+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:33:32.020+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.132 seconds
[2024-10-18T05:34:02.688+0000] {processor.py:186} INFO - Started process (PID=7045) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:34:02.690+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:34:02.693+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:34:02.692+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:34:02.731+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:34:02.722+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:34:02.734+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:34:02.782+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.113 seconds
[2024-10-18T05:34:33.056+0000] {processor.py:186} INFO - Started process (PID=7056) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:34:33.059+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:34:33.066+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:34:33.065+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:34:33.138+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:34:33.124+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:34:33.143+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:34:33.210+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.186 seconds
[2024-10-18T05:35:03.510+0000] {processor.py:186} INFO - Started process (PID=7067) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:35:03.513+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:35:03.518+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:35:03.517+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:35:03.584+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:35:03.569+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:35:03.589+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:35:03.680+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.194 seconds
[2024-10-18T05:35:33.873+0000] {processor.py:186} INFO - Started process (PID=7079) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:35:33.874+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:35:33.877+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:35:33.876+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:35:33.904+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:35:33.897+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:35:33.907+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:35:33.944+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.080 seconds
[2024-10-18T05:36:04.868+0000] {processor.py:186} INFO - Started process (PID=7090) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:36:04.870+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:36:04.873+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:36:04.873+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:36:04.910+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:36:04.903+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:36:04.913+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:36:04.996+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.149 seconds
[2024-10-18T05:36:35.250+0000] {processor.py:186} INFO - Started process (PID=7101) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:36:35.252+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:36:35.255+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:36:35.254+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:36:35.287+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:36:35.277+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:36:35.290+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:36:35.327+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.088 seconds
[2024-10-18T05:37:05.578+0000] {processor.py:186} INFO - Started process (PID=7113) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:37:05.581+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:37:05.592+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:37:05.591+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:37:05.655+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:37:05.642+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:37:05.659+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:37:05.736+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.188 seconds
[2024-10-18T05:37:36.602+0000] {processor.py:186} INFO - Started process (PID=7124) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:37:36.604+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:37:36.609+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:37:36.608+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:37:36.658+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:37:36.652+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:37:36.663+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:37:36.730+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.150 seconds
[2024-10-18T05:38:06.970+0000] {processor.py:186} INFO - Started process (PID=7135) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:38:06.972+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:38:06.987+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:38:06.986+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:38:07.040+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:38:07.025+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:38:07.052+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:38:07.151+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.228 seconds
[2024-10-18T05:38:37.675+0000] {processor.py:186} INFO - Started process (PID=7146) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:38:37.688+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:38:37.693+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:38:37.692+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:38:37.816+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:38:37.789+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:38:37.820+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:38:38.003+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.359 seconds
[2024-10-18T05:39:08.751+0000] {processor.py:186} INFO - Started process (PID=7157) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:39:08.754+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:39:08.765+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:39:08.765+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:39:08.801+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:39:08.792+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:39:08.804+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:39:08.853+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.125 seconds
[2024-10-18T05:39:39.175+0000] {processor.py:186} INFO - Started process (PID=7169) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:39:39.178+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:39:39.183+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:39:39.182+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:39:39.217+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:39:39.210+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:39:39.219+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:39:39.251+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.099 seconds
[2024-10-18T05:40:09.583+0000] {processor.py:186} INFO - Started process (PID=7180) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:40:09.584+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:40:09.587+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:40:09.586+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:40:09.617+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:40:09.610+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:40:09.619+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:40:09.650+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.079 seconds
[2024-10-18T05:40:39.810+0000] {processor.py:186} INFO - Started process (PID=7191) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:40:39.812+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:40:39.815+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:40:39.815+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:40:39.835+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:40:39.831+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:40:39.838+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:40:39.870+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.072 seconds
[2024-10-18T05:41:10.432+0000] {processor.py:186} INFO - Started process (PID=7202) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:41:10.436+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:41:10.440+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:41:10.439+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:41:10.492+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:41:10.480+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:41:10.496+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:41:10.568+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.159 seconds
[2024-10-18T05:41:41.301+0000] {processor.py:186} INFO - Started process (PID=7213) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:41:41.303+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:41:41.306+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:41:41.305+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:41:41.339+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:41:41.332+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:41:41.342+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:41:41.383+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.097 seconds
[2024-10-18T05:42:12.170+0000] {processor.py:186} INFO - Started process (PID=7226) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:42:12.171+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:42:12.173+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:42:12.173+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:42:12.194+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:42:12.189+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:42:12.196+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:42:12.224+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.065 seconds
[2024-10-18T05:42:42.597+0000] {processor.py:186} INFO - Started process (PID=7237) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:42:42.599+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:42:42.602+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:42:42.601+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:42:42.624+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:42:42.620+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:42:42.627+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:42:42.666+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.078 seconds
[2024-10-18T05:43:13.074+0000] {processor.py:186} INFO - Started process (PID=7248) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:43:13.075+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:43:13.078+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:43:13.077+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:43:13.101+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:43:13.095+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:43:13.103+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:43:13.135+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.073 seconds
[2024-10-18T05:43:43.511+0000] {processor.py:186} INFO - Started process (PID=7259) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:43:43.514+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:43:43.527+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:43:43.517+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:43:43.553+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:43:43.548+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:43:43.559+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:43:43.619+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.159 seconds
[2024-10-18T05:44:13.885+0000] {processor.py:186} INFO - Started process (PID=7269) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:44:13.887+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:44:13.890+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:44:13.889+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:44:13.918+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:44:13.910+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:44:13.921+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:44:13.976+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.105 seconds
[2024-10-18T05:44:44.197+0000] {processor.py:186} INFO - Started process (PID=7280) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:44:44.206+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:44:44.214+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:44:44.212+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:44:44.281+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:44:44.263+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:44:44.298+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:44:44.344+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.202 seconds
[2024-10-18T05:45:14.576+0000] {processor.py:186} INFO - Started process (PID=7291) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:45:14.579+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:45:14.583+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:45:14.582+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:45:14.637+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:45:14.624+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:45:14.642+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:45:14.704+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.160 seconds
[2024-10-18T05:45:45.010+0000] {processor.py:186} INFO - Started process (PID=7302) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:45:45.016+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:45:45.024+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:45:45.023+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:45:45.108+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:45:45.088+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:45:45.116+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:45:45.199+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.225 seconds
[2024-10-18T05:46:15.322+0000] {processor.py:186} INFO - Started process (PID=7313) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:46:15.324+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:46:15.337+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:46:15.337+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:46:15.368+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:46:15.362+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:46:15.371+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:46:15.446+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.150 seconds
[2024-10-18T05:46:45.739+0000] {processor.py:186} INFO - Started process (PID=7324) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:46:45.742+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:46:45.744+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:46:45.744+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:46:45.783+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:46:45.771+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:46:45.786+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:46:45.830+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.106 seconds
[2024-10-18T05:47:16.471+0000] {processor.py:186} INFO - Started process (PID=7334) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:47:16.473+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:47:16.475+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:47:16.474+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:47:16.496+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:47:16.491+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:47:16.498+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:47:16.535+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.073 seconds
[2024-10-18T05:47:47.221+0000] {processor.py:186} INFO - Started process (PID=7345) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:47:47.237+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:47:47.242+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:47:47.242+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:47:47.297+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:47:47.287+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:47:47.301+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:47:47.410+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.211 seconds
[2024-10-18T05:48:17.998+0000] {processor.py:186} INFO - Started process (PID=7354) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:48:18.001+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:48:18.005+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:48:18.003+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:48:18.057+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:48:18.049+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:48:18.060+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:48:18.095+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.115 seconds
[2024-10-18T05:48:48.537+0000] {processor.py:186} INFO - Started process (PID=7365) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:48:48.539+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:48:48.553+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:48:48.552+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:48:48.660+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:48:48.649+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:48:48.669+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:48:48.744+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.230 seconds
[2024-10-18T05:49:19.089+0000] {processor.py:186} INFO - Started process (PID=7375) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:49:19.091+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:49:19.093+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:49:19.093+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:49:19.119+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:49:19.113+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:49:19.121+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:49:19.156+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.077 seconds
[2024-10-18T05:49:49.518+0000] {processor.py:186} INFO - Started process (PID=7385) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:49:49.520+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:49:49.523+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:49:49.522+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:49:49.545+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:49:49.540+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:49:49.547+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:49:49.582+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.078 seconds
[2024-10-18T05:50:19.765+0000] {processor.py:186} INFO - Started process (PID=7397) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:50:19.767+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:50:19.770+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:50:19.770+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:50:19.802+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:50:19.795+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:50:19.804+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:50:19.842+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.089 seconds
[2024-10-18T05:50:50.886+0000] {processor.py:186} INFO - Started process (PID=7408) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:50:50.888+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:50:50.891+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:50:50.890+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:50:50.919+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:50:50.909+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:50:50.921+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:50:50.977+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.101 seconds
[2024-10-18T05:51:21.505+0000] {processor.py:186} INFO - Started process (PID=7420) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:51:21.506+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:51:21.509+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:51:21.508+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:51:21.538+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:51:21.528+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:51:21.540+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:51:21.582+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.097 seconds
[2024-10-18T05:51:52.220+0000] {processor.py:186} INFO - Started process (PID=7431) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:51:52.222+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:51:52.226+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:51:52.225+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:51:52.264+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:51:52.255+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:51:52.268+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:51:52.321+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.133 seconds
[2024-10-18T05:52:22.567+0000] {processor.py:186} INFO - Started process (PID=7442) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:52:22.580+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:52:22.584+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:52:22.583+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:52:22.626+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:52:22.620+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:52:22.628+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:52:22.707+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.162 seconds
[2024-10-18T05:52:53.863+0000] {processor.py:186} INFO - Started process (PID=7453) to work on /opt/airflow/dags/bas.py
[2024-10-18T05:52:53.866+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bas.py for tasks to queue
[2024-10-18T05:52:53.870+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:52:53.869+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/bas.py
[2024-10-18T05:52:53.904+0000] {logging_mixin.py:190} INFO - [2024-10-18T05:52:53.898+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/bas.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bas.py", line 2, in <module>
    import wget
ModuleNotFoundError: No module named 'wget'
[2024-10-18T05:52:53.906+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bas.py
[2024-10-18T05:52:53.996+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bas.py took 0.165 seconds
